src/parser/jobs/nin/modules/Positionals.ts

```diff
@@ -1,15 +1,24 @@
 import {PotencyModifier, Positionals as CorePositionals} from 'parser/core/modules/Positionals'
 
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+const ARMOR_CRUSH_BASE_POTENCY = 140
+const ARMOR_CRUSH_COMBO_POTENCY_6_0 = 340
+const ARMOR_CRUSH_COMBO_POTENCY_6_1 = 360
+const AEOLIAN_EDGE_BASE_POTENCY = 140
+const AEOLIAN_EDGE_COMBO_POTENCY_6_0 = 360
+const AEOLIAN_EDGE_COMBO_POTENCY_6_1 = 380
+/* eslint-enable @typescript-eslint/no-magic-numbers */
+
 export class Positionals extends CorePositionals {
 	positionals = [{
 		action: this.data.actions.ARMOR_CRUSH,
 		potencies: [
 			{
-				value: 140,
+				value: ARMOR_CRUSH_BASE_POTENCY,
```

Comment:
Prefer to get the base potency from the data (so `this.data.actions.AEOLIAN_EDGE.potency`, see SAM for an example) and keep the base potency in the data.

Reply:
Resolved in [95e3207](https://github.com/xivanalysis/xivanalysis/pull/1704/commits/95e3207223d677ad20b684a6d8e0d21b6e397510)

Reply:
This does not appear to be resolved.